### DISCUSSION TITLE
ability to change default suffix

### DISCUSSION THREAD

**Discussion by travisedson** [ieatacid/OctoPrint-GcodeEditor#39] [Ideas]
hello this is a feature request, it is not critical but, i would really appreciate having the ability to change the default suffix for the modified file

thank you for your addon

best
travis

**Comment by ieatacid** [comment]
The g/gco/gcode file extensions are [enforced by OctoPrint](https://github.com/OctoPrint/OctoPrint/blob/0fcb15d875c504f3186dd11496d7b9ef77afca3b/src/octoprint/filemanager/__init__.py#L29).

**Reply by travisedson** [reply]
that's not what i'm talking about. that is the file extension. I'm talking about the _modified suffix that is the default from the addon. it would be nice to be able to change this to whatever I wanted to.

**Reply by ieatacid** [reply]
I removed the behavior of the plugin inserting "_modified" to the end of the file name. One guy wants it to replace the original file, you want to change "_modified" to whatever you want. Now it defaults to the original file name (as it always did before a merged pull request changed that). So leaving the name as-is will replace the original file, or you can change the file name to whatever you want.

**Reply by travisedson** [reply]
Better. Thanks.

**Reply by travisedson** [reply]
I know it's anal-rententive. I am _revising_, not modifying. Again, thanks for the addon, it's very useful.
